docs/source/index.rst

```diff
@@ -143,7 +315,7 @@ messy situations in everyday problems.
 .. toctree::
    :maxdepth: 1
    :hidden:
-   :caption: Fundamentals
+   :caption: API
```

Comment:
Maybe "Dask APIs"? Or something else to distinguish from the API reference lower down?

Reply:
Hrm, I agree with the concern (mildly agree at least).  I'm not a fan of this particular solution though.